ISM43362/ISM43362.cpp

```diff
@@ -218,6 +273,18 @@ int ISM43362::connect(const char *ap, const char *passPhrase, ism_security_t ap_
         return NSAPI_ERROR_PARAMETER;
     }
 
+     /* Check country code is acceptable */
+    bool ret = check_country_code(MBED_CONF_ISM43362_WIFI_COUNTRY_CODE);
```

Comment:
To allow backward compatibility shouldn't support the standard case where MBED_CONF_ISM43362_WIFI_COUNTRY_CODE is not defined ?
Like
#ifdef MBED_CONF_ISM43362_WIFI_COUNTRY_CODE
      ... all related code here ...
#endif

Reply:
The MBED_CONF_ISM43362_WIFI_COUNTRY_CODE is always defined.
Indeed, the flag is defined by default to JP in mbed_app.json file of wifi-ism43362 module.